src/translations/nb.ts

```diff
@@ -145,5 +145,29 @@ const translations = {
     'Vi klarte ikke å hente pensjonsavtalene dine fra Norsk Pensjon. Prøv igjen senere.',
   'grunnlag.pensjonsavtaler.tabell.title.left': 'Pensjonsavtaler',
   'grunnlag.pensjonsavtaler.tabell.title.right': 'Årlig beløp',
+  'landingsside.for.deg.foedt.foer.1963': 'For deg født før 1963',
+  'landingsside.du.maa.bruke.detaljert':
+    'Du må bruke vår detaljerte kalkulator for å beregne din pensjon.',
+  'landingsside.for.deg.foedt.etter.1963': 'For deg født i 1963 eller senere',
+  'landingsside.velge-mellom-detaljert-og-enkel':
+    'Du kan velge mellom enkel eller detaljert kalkulator. Enkel kalkulator passer for deg som vil ha en rask oversikt. Detaljert kalkulator passer for deg som vil ha en mer spesifisert beregning. Enkel kalkulator er under utvikling. Derfor må du bruke detaljert kalkulator hvis du: ',
+  'landingsside.liste.1':
+    'har bodd eller jobbet utenfor Norge i mer enn 5 år etter fylte 16 år',
+
+  'landingsside.liste.2': 'mottar uføretrygd eller gjenlevendepensjon',
+  'landingsside.liste.3':
+    'har hatt betydelig endring i inntekt de siste 2 årene',
+  'landingsside.liste.4': 'har særaldersgrense',
+  'landingsside.button.detaljert-kalkulator-utlogget':
```

Comment:
har brukt '_' hittil og ikke '-' som separator i keys.
Ikke viktig hvilken av de to, bare at det er konsekvent den ene eller den andre